### PR TITLE
New term methicillin-susceptible staphylococcus aureus

### LINKED ISSUE #7835: Request for new term "methicillin-susceptible staphylococcus aureus infectious disease"
### Label

methicillin-susceptible staphylococcus aureus infectious disease

### Synonyms

_No response_

### Synonym type

None

### Definition

Infections caused by a strain of Staphylococcus aureus that is susceptible to the action of the antibiotic methicillin.

### Parent term

MONDO:0005545

### Children term(s)

_No response_

### ORCID Identifier

_No response_

### Website URL

_No response_

### Additional comments

_No response_

### PR DESCRIPTION
**Pull request by yshwetar**
Closes #7835

- Definition
- Pattern with link
- PMID
- ORCID
- Term tracker

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -579746,6 +579746,15 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-18T18:06:40Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7858" xsd:anyURI
 
+[Term]
+id: MONDO:1040020
+name: methicillin-susceptible staphylococcus aureus infectious disease
+def: "A bacterial disease caused by methicillin-susceptible staphylococcus aureus." [https://orcid.org/0000-0002-0587-4693, MONDO:patterns/infectious_disease_by_agent, PMID:33144192]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:579752]: @yshwetar please update the beginning to "a bacterial **infectious** disease".
Otherwise, the rest is great.

**Review by sabrinatoro — APPROVED** [review]
@yshwetar There is a minor change being requested. 
I am approving this issue as everything else looks good. 
Feel free to merge after this change+QC checks

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo]: done

Comment (yshwetar) [reply src/ontology/mondo-edit.obo]: Looks like I'm too slow to respond! Thanks Sabrina :)

Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo]: I was just there, I thought I would just do it :-) nothing to do with your speed :-)

**Pull request merged**